e2e-tests/tfm/cypress/e2e/pages/utilisation-reports/utilisationReportAddToAnExistingPaymentPage.js

```diff
@@ -1,9 +1,11 @@
 const utilisationReportAddToAnExistingPaymentPage = {
   selectedReportedFeesDetailsTable: () => cy.contains('table', 'Selected reported fees details'),
-  availablePaymentGroups: () => cy.get('[data-cy="payment-group-radio-input"]'),
+  availablePaymentGroups: () => cy.get('[data-cy="available-payment-groups"]'),
   continueButton: () => cy.get('[data-cy="continue-button"]'),
   paymentGroupRadioButton: (paymentGroupId) => cy.get(`input[id="${paymentGroupId}"]`),
   errorSummary: () => cy.get('[data-cy="error-summary"]'),
+  backLink: () => cy.get('[data-cy="back-link"]'),
+  cancelLink: () => cy.get('[data-cy="cancel-link"]'),
```

Comment:
Not for this PR, these back/cancel links are repeated throughout the E2E tests, want to move these into partials and introduce commands like `cy.clickBackLink();`